avm/src/vm/opcode.rs

```diff
@@ -222,2012 +224,2014 @@ pub static OPCODES: Lazy<HashMap<i64, ByteOpcode>> = Lazy::new(|| {
 
   // Type conversion opcodes
   cpu!(i8f64 => fn(args, hand_mem) {
-    let out = hand_mem.read_fixed(args[0]) as f64;
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    let out = hand_mem.read_fixed(args[0])? as f64;
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
   });
   cpu!(i16f64 => fn(args, hand_mem) {
-    let out = hand_mem.read_fixed(args[0]) as f64;
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    let out = hand_mem.read_fixed(args[0])? as f64;
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
   });
   cpu!(i32f64 => fn(args, hand_mem) {
-    let out = hand_mem.read_fixed(args[0]) as f64;
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    let out = hand_mem.read_fixed(args[0])? as f64;
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
   });
   cpu!(i64f64 => fn(args, hand_mem) {
-    let out = hand_mem.read_fixed(args[0]) as f64;
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    let out = hand_mem.read_fixed(args[0])? as f64;
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
   });
   cpu!(f32f64 => fn(args, hand_mem) {
-    let out = f32::from_ne_bytes((hand_mem.read_fixed(args[0]) as i32).to_ne_bytes());
-    hand_mem.write_fixed(args[2], i32::from_ne_bytes(out.to_ne_bytes()) as i64);
-    None
+    let out = f32::from_ne_bytes((hand_mem.read_fixed(args[0])? as i32).to_ne_bytes());
+    hand_mem.write_fixed(args[2], i32::from_ne_bytes(out.to_ne_bytes()) as i64)?;
+    Ok(None)
   });
   cpu!(strf64 => fn(args, hand_mem) {
-    let s = HandlerMemory::fractal_to_string(hand_mem.read_fractal(args[0]));
+    let s = HandlerMemory::fractal_to_string(hand_mem.read_fractal(args[0])?)?;
     let out: f64 = s.parse().unwrap();
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
   });
   cpu!(boolf64 => fn(args, hand_mem) {
-    let out = hand_mem.read_fixed(args[0]) as f64;
-    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()));
-    None
+    let out = hand_mem.read_fixed(args[0])? as f64;
+    hand_mem.write_fixed(args[2], i64::from_ne_bytes(out.to_ne_bytes()))?;
+    Ok(None)
```

Comment:
These opcodes sound very confused and unsure of themselves "read fixed argument 0? I guess? Then write it as a bool to argument 2? Ok, nothing else."